src/main/java/uk/ac/ebi/ega/accession/configuration/security/DisableSecurityConfig.java

```diff
@@ -0,0 +1,35 @@
+/*
+ *
+ * Copyright 2019 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package uk.ac.ebi.ega.accession.configuration.security;
+
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.oauth2.config.annotation.web.configuration.EnableResourceServer;
+import org.springframework.security.oauth2.config.annotation.web.configuration.ResourceServerConfigurerAdapter;
+
+@ConditionalOnProperty(value = "security.enabled", havingValue = "false")
+@Configuration
+@EnableResourceServer
+public class DisableSecurityConfig extends ResourceServerConfigurerAdapter {
```

Comment:
What about using the property described [here](https://stackoverflow.com/a/34150562)?

Reply:
it is kind of deprecated I guess, that is used this config. https://spring.io/blog/2017/09/15/security-changes-in-spring-boot-2-0-m4

Reply:
It is deprecated and also Oauth has gone a massive revamp, so, unfortunately in the future we will need to do some adjustments, as right now moving to that revamp would require us to up all the versions.